sktime/classification/interval_based/_tsf.py

```diff
@@ -128,37 +214,23 @@ def fit(self, X, y):
             self.n_intervals = 1
         if self.series_length < self.min_interval:
             self.min_interval = self.series_length
-        self.intervals = np.zeros((self.n_estimators, self.n_intervals, 2),
-                                  dtype=int)
-        for i in range(self.n_estimators):
-            transformed_x = np.empty(shape=(3 * self.n_intervals, n_instances))
-            # Find the random intervals for classifier i and concatentate
-            # features
-            for j in range(self.n_intervals):
-                self.intervals[i][j][0] = rng.randint(
-                    self.series_length - self.min_interval)
-                length = rng.randint(
-                    self.series_length - self.intervals[i][j][0] - 1)
-                if length < self.min_interval:
-                    length = self.min_interval
-                self.intervals[i][j][1] = self.intervals[i][j][0] + length
-                # Transforms here, just hard coding it, so not configurable
-                means = np.mean(
-                    X[:, self.intervals[i][j][0]:self.intervals[i][j][1]],
-                    axis=1)
-                std_dev = np.std(
-                    X[:, self.intervals[i][j][0]:self.intervals[i][j][1]],
-                    axis=1)
-                slope = self._lsq_fit(
-                    X[:, self.intervals[i][j][0]:self.intervals[i][j][1]])
-                transformed_x[3 * j] = means
-                transformed_x[3 * j + 1] = std_dev
-                transformed_x[3 * j + 2] = slope
-            tree = clone(self.base_estimator)
-            tree.set_params(**{"random_state": self.random_state})
-            transformed_x = transformed_x.T
-            tree.fit(transformed_x, y)
-            self.classifiers.append(tree)
+
+        self.intervals = [
```

Comment:
Could we add trailing underscores to variables generated in fit following sklearn's convention, so 

```suggestion
        self.intervals_ = [
```

Same for classifiers below